src/geodesic.rs

```diff
@@ -338,15 +338,14 @@ impl Geodesic {
             calp2 = sbet12
                 - cbet1
                     * sbet2
-                    * (geomath::sq(somg12)
```

Comment:
I'll review this more carefully after you answer my other question, but I'd bet money on this being a mistranslation of a python ternary. Ran into a couple of those myself. 😉

Reply:
That matches with my best guess about this. In short, though, the old version didn't quite match Karney's code, so I changed it to match.